Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1181,6 +1181,14 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="submission_accept_button">"Einverstanden"</string>
     <!-- XACT: other warning - illustration description, explanation image -->
     <string name="submission_positive_other_illustration_description">"Ein Smartphone übermittelt einen positiven Testbefund verschlüsselt ins System."</string>
+    <!-- XHED: CLose screen popup title for no consent given cancel dialog -->
+    <string name="submission_positive_other_warning_dialog_title">"Zufallskennungen mit der App „Corona-Warn“ teilen?"</string>
```

Comment:
Throughout the app, we use "Zufalls-IDs" and not "Zufallskennungen", this should be changed.
Also, I think it would be better to rephrase the title as: "Zufalls-IDs über Corona-Warn-App teilen?"

Reply:
Alright, I've updated it with the new text.